docs/resources/feature_state.md

```diff
@@ -88,12 +88,15 @@ resource "flagsmith_feature_state" "feature_1_dev_segment_override" {
 <a id="nestedatt--feature_state_value"></a>
 ### Nested Schema for `feature_state_value`
 
+Required:
+
+- `type` (String) Type of the feature state value, can be `unicode`, `int` or `bool`
+
 Optional:
```

Comment:
Hmm... I guess it's slightly confusing that these are optional but I think that at least one of them is required?

Reply:
Yes, that is correct, although we mention that at a couple of places, should be fine I think